src/univalue/test/unitester.cpp

```diff
@@ -6,13 +6,15 @@
 
 #include <cassert>
 #include <cstdio>
+#include <cstdlib>
 #include <string>
 
-#ifndef JSON_TEST_SRC
-#error JSON_TEST_SRC must point to test source directory
-#endif
-
-std::string srcdir(JSON_TEST_SRC);
+std::string srcdir = []() {
+    if (const char* env_srcdir = std::getenv("UNIVALUE_JSON_TEST_SRC")) {
+        return env_srcdir;
+    }
+    return ".";
```

Comment:
Maybe call `exit(1)` directly? Or breaking this out so that `main()` returns non-zero in this case? Otherwise there's no (helpful) indication of how to run the binary correctly when it fails.

Reply:
> A step further could even go as far as directly embedding the blobs in the exe?

I think I would prefer that.